app/mapper/UserMapper.py

```diff
@@ -9,9 +8,8 @@ def __init__():
     pass
 
 
-def makeNew(name, password):
-    user = User(name, password)
```

Comment:
it for sure wasn't. They just wrote it to pass minimal code review from the prof